src/components/application_manager/src/hmi_state.cc

```diff
@@ -227,7 +227,8 @@ mobile_apis::HMILevel::eType AudioSource::hmi_level() const {
   // Should be investigated (used in multiple places here), since looks weird
   if (Compare<HMILevel::eType, EQ, ONE>(parent()->hmi_level(),
                                         HMILevel::HMI_BACKGROUND,
-                                        HMILevel::HMI_NONE)) {
+                                        HMILevel::HMI_NONE,
+                                        HMILevel::HMI_FULL)) {
```

Comment:
why need this change?

Reply:
@yang1070 This is a fix for the case when a media remote control app is in FULL and it sends a SetInteriorVehicleData to change the audio source to Radio and has keepContext=true, HMI sends AUDIO_SOURCE event to core, SDL shall keep the app’s HMI in **FULL** and set audioStreamingState to NOT_AUDIBLE.
(before SDL brings app from FULL state to BACKGROUND)

Reply:
@LitvinenkoIra Got it. Thanks,